Problems/1712/1712.py

```diff
@@ -10,10 +10,9 @@
 #expense = a + b*x_input
 #profit = c*x_input 2100000000
 
-for x_input in range(2100000000):
-    if a + b*x_input < c*x_input:
-        print(x_input)
-        break
-    if x_input == 2100000000:
-        print(-1)
-        break 
\ No newline at end of file
+
+if b>=c:
+    print(-1)
+else:
+    print(int(a/(c-b))+1)
```

Comment:
```suggestion
    print((a//(c-b))+1)
```
별거 아닌 것 같아도, type casting이 들어가게 되면 읽는 사람으로 하여금 그 안에 내용물이 원래 무엇이었는지
생각해보게 만드는 듯. 이미 a,b,c가 모두 integer 인 이상 불필요.

Reply:
네 불필요한 code를 지우는 습관을 들이겠습니다

Reply:
이거 첨에 불필요하게 for문을 겁나 많이 돌려서 run time error 뜬다고 말했던거 같네요